ultravox/tools/ds_tool/ds_tool.py

```diff
@@ -36,14 +41,29 @@ def __post_init__(self):
 
     def map_split(self, ds_split: datasets.Dataset, num_proc: int) -> datasets.Dataset:
         print(f'TTS mapping "{self.column_name}" to "{self.audio_column_name}"...')
-        return ds_split.map(self._map_sample, num_proc=num_proc).cast_column(
+        return ds_split.map(
+            self._map_sample, num_proc=num_proc, writer_batch_size=self.write_batch_size
+        ).cast_column(
             self.audio_column_name, datasets.Audio(sampling_rate=self.sample_rate)
         )
 
     def _map_sample(self, sample):
+        for field in self.format_fields:
+            sample[field] = format_asr_text(sample[field])
         # using a Jinja template for some added flexibility
         # The {{ var }} syntax is how Jinja denotes variables
         text = jinja2.Template("{{" + self.column_name + "}}").render(**sample)
```

Comment:
is this line still needed?

Reply:
I'm still confused. Why do you do the same thing twice?
![image](https://github.com/user-attachments/assets/1fed8003-4c2c-4392-ad41-dfcbdef1442f)

Reply:
good catch, this is an error.